src/OrchardCore.Modules/OrchardCore.OpenId/Settings/OpenIdValidationSettings.cs

```diff
@@ -8,5 +8,8 @@ public class OpenIdValidationSettings
         public Uri Authority { get; set; }
         public bool DisableTokenTypeValidation { get; set; }
         public string Tenant { get; set; }
+#nullable enable
```

Comment:
I'm all for using nullable references but I'm not sure what's the policy in place in OC. I'm not so sure it's worth decorating only select properties in a file.

/cc @sebastienros